roles/variables/vars/main.yml

```diff
@@ -144,7 +144,7 @@ kafka_controller_properties:
       confluent.ansible.managed: 'true'
       confluent.license.topic: _confluent-command
       confluent.license.topic.replication.factor: "{{kafka_controller_default_internal_replication_factor}}"
-      confluent.metadata.topic.replication.factor: "{{kafka_controller_default_internal_replication_factor}}"
+      confluent.metadata.topic.replication.factor: "{{kafka_broker_default_internal_replication_factor}}"
```

Comment:
Why are we using kafka_broker properties here

Reply:
since MDS runs on broker and not controller

Reply:
will test and exclude if needed